### DISCUSSION TITLE
Looking for servers to run github runner for nightly build CI (arm64/32)

### DISCUSSION THREAD

**Discussion by rustdesk** [rustdesk/rustdesk#5450] [General]
Lately, our github nightly build CI failed on arm every day because of insufficient memory.

https://github.com/rustdesk/rustdesk/actions/runs/5911062641

If the server is amd64, we need big memory (>20G) to run the qemu for building arm64/arm32.
If the server is arm64 or arm32, 8G memory is OK

**Comment by deep-soft** [comment]
> our github nightly build CI failed on arm every day

that's because recent changes in `.github/workflows/flutter-build.yml` ?!

I use previous CI for aarch64-unknown-linux-gnu and build isn't failing.

**Reply by rustdesk** [reply]
I changed it because of failure, but no luck.

**Reply by deep-soft** [reply]
I have created new branch from rustdesk/rustdesk to apply only aarch64 build.
It took "only" 6 hours to complete.
Until github offers native arm VM's, the only solution is self hosted arm native VM.
:(

**Comment by mikekgr** [comment]
> 

Dear @rustdesk ,
I have a VM that is aarch64 based (Oracle Frankfurt cloud) with 20GB RAM and it is real fast. Do you want me to give you SSH access ?
O/S: Canonical-Ubuntu-22.04-aarch64
Best Regards,
Mike Kranidis

**Reply by rustdesk** [reply]
@mikekgr Please. ###

Thank you very much.

**Reply by mikekgr** [reply]
See your emails ...
